config/checkstyle_input_checks.xml

```diff
@@ -19,12 +19,11 @@
     <!-- Check to make sure check UT input files have config and trailing comments -->
     <module name="Regexp">
       <property name="id" value="ConfigCommentOnTopInputs"/>
-      <property name="format"
-               value="(\* Config)"/>
+      <property name="format" value="(\*)"/>
```

Comment:
it seems to ambiguous
There is example of `file starts with` here https://checkstyle.org/config_regexp.html#Regexp_Examples
smth like `(\* Config|\A\\\*\n)`

Reply:
done